dietpi/dietpi-software

```diff
@@ -3435,6 +3441,87 @@ _EOF_
 
 		fi
 
+		software_id=99 # Prometheus Node Exporter
+		if (( ${aSOFTWARE_INSTALL_STATE[$software_id]} == 1 )); then
+
+			Banner_Installing
+
+			# Use the default port
+			local port="9100"
+
+			local arch=''
+			case $G_HW_ARCH_NAME in
+				'armv6l') arch='armv6';;
+				'armv7l') arch='armv7';;
+				'aarch64') arch='arm64';;
+				'x86_64') arch='amd64';;
+			esac
+
+			# Reinstall: Remove previous instances
+			[[ -e '/usr/local/bin/node_exporter' ]] && G_EXEC rm -R /usr/local/bin/node_exporter
+			[[ -e '/usr/local/bin/raspberrypi_exporter' ]] && G_EXEC rm -R /usr/local/bin/raspberrypi_exporter
+
```

Comment:
The ne files overwrite the old ones, isn't it? So this explicit removal shouldn't be required, and skipping it better covers the case where the download fails, to not leave users without any executable:
```suggestion
```